alica_engine/include/engine/PlanInterface.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
```

Comment:
Seems this file is not used in this pr, so push it back to be reviewed in the pr where it is used